plugins/block-dynamic-connection/src/dynamic_if.ts

```diff
@@ -109,18 +102,17 @@ Blockly.Blocks['dynamic_if'] = {
       this.appendStatementInput('ELSE')
           .appendField(Blockly.Msg['CONTROLS_IF_MSG_ELSE'], 'else');
     }
-    const next = parseInt(xmlElement.getAttribute('next'));
+    const next = parseInt(xmlElement.getAttribute('next')!);
```

Comment:
In addition to the non-null assertion lint, I think this should probably have a radix specified (and perhaps also default values; see earlier comment about `.getAttribute`/`parseInt`):
```suggestion
    const next = parseInt(xmlElement.getAttribute('next')!, 10);
```

Reply:
Done.